src/include/utils.h

```diff
@@ -68,6 +68,8 @@ inline ncclResult_t getRandomData(void* buffer, size_t bytes) {
   return ret;
 }
 
+bool ncclNeedEnableContextTrack(int cuDeviceId);
```

Comment:
```suggestion
bool rcclNeedEnableContextTrack(int cuDeviceId);
```
This is not a `nccl` function, we should use the `rccl` naming convention.